### PR TITLE
Document running nodes remotely (related to ENT-1678)

### PR DESCRIPTION
**Pull request by szymonsztuka**
Document running nodes on a remote machine after generating them on a local machine with Cordform task.

### CODE REVIEW

File: docs/source/running-a-node.rst [hunk 1]
```diff
@@ -104,3 +106,31 @@ After the nodes are started up, you can use ``docker ps`` command to see how the
    (Community Edition) is enough. Please refer to `Docker CE documentation <https://www.docker.com/community-edition>`_
    and `Docker Compose documentation <https://docs.docker.com/compose/install/>`_ for installation instructions for all
    major operating systems.
+
+Starting all nodes at once on a remote machine from the command line
+--------------------------------------------------------------------
+
+Corda Gradle plugin ``Cordform`` allows you to generate a set of node on the same machine where they are run.
+In some cases this may be not feasible as a machine to run the nodes may not have access to Maven repositories or may not be a development environment.
```
Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: I think you can drop this sentence.

File: docs/source/running-a-node.rst [hunk 2]
```diff
@@ -104,3 +106,31 @@ After the nodes are started up, you can use ``docker ps`` command to see how the
    (Community Edition) is enough. Please refer to `Docker CE documentation <https://www.docker.com/community-edition>`_
    and `Docker Compose documentation <https://docs.docker.com/compose/install/>`_ for installation instructions for all
    major operating systems.
+
+Starting all nodes at once on a remote machine from the command line
+--------------------------------------------------------------------
+
+Corda Gradle plugin ``Cordform`` allows you to generate a set of node on the same machine where they are run.
```
Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: I'd change this to: "By default, ``Cordform`` expects the nodes it generates to be run on the same machine where they were generated."

File: docs/source/running-a-node.rst [hunk 3]
```diff
@@ -104,3 +106,31 @@ After the nodes are started up, you can use ``docker ps`` command to see how the
...
+In some cases this may be not feasible as a machine to run the nodes may not have access to Maven repositories or may not be a development environment.
+In order to run the nodes remotely, the nodes can be deployed locally and then copied to a remote server.
+Additional steps may be required in case generated addresses are not localhost or a specific IP needs to be used on the remote machine.
```
Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: I don't understand what you mean here? In what cases would this be an issue?

File: docs/source/running-a-node.rst (hunk 1 above)
Comment (szymonsztuka) [reply docs/source/running-a-node.rst]: Dropped

File: docs/source/running-a-node.rst (hunk 2 above)
Comment (szymonsztuka) [reply docs/source/running-a-node.rst]: Done

File: docs/source/running-a-node.rst (hunk 3 above)
Comment (szymonsztuka) [reply docs/source/running-a-node.rst]: Rephrase that sentence.

File: docs/source/running-a-node.rst [hunk 4]
```diff
@@ -104,3 +106,30 @@ After the nodes are started up, you can use ``docker ps`` command to see how the
    (Community Edition) is enough. Please refer to `Docker CE documentation <https://www.docker.com/community-edition>`_
    and `Docker Compose documentation <https://docs.docker.com/compose/install/>`_ for installation instructions for all
    major operating systems.
+
+Starting all nodes at once on a remote machine from the command line
+--------------------------------------------------------------------
+
+By default, ``Cordform`` expects the nodes it generates to be run on the same machine where they were generated.
+In order to run the nodes remotely, the nodes can be deployed locally and then copied to a remote server.
+Additional steps may be required in case generated addresses in the node configuration are incorrect for the remote machine (e.g. ``localhost`` cannot be used).
```
Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: How about changing this sentence to:

"If after copying the nodes to the remote machine you encounter errors related to ``localhost`` resolution, you will additionally need to follow the steps below."

**Pull request merged**